src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -107,6 +107,10 @@ public Reporter getReporter() {
         return reporter;
     }
 
+    public void setReporter(Reporter reporter) {
```

Comment:
Here you're breaking in a way the way the reporter was at first designed... It sure is handy to put it in LfNetwork but it was first thought to be passed along all the methods and not retained, in the javadoc:

> A reporter is not meant to be shared with other threads nor to be saved as a class parameter

Reply:
A LF network won't ever be shared between multiple thread as it is used as working data (including writing working data)